database.json

```diff
@@ -2214,6 +2214,12 @@
     "repo": "resulty",
     "desc": "Provides simple, Rust-like Result and Option objects for Deno."
   },
+  "rhinoder": {
+    "type": "github",
+    "owner": "Caesar2011",
+    "repo": "rhinoder",
+    "desc": "Reloads a deno application on every file change"
+  },
```

Comment:
put the repo in an alphabetic manner

i.e. after the ripemd160 module & before the router module.

that will solve the CI error.

Reply:
But `h` comes before `i` in the alphabet? I am so confused right now!

Reply:
ooh sorry just put in right place https://github.com/denoland/deno_website2/pull/882/checks?check_run_id=717111435